app/sidekiq/mhv/account_creator_job.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+require 'mhv/user_account/creator'
+
+module MHV
+  class AccountCreatorJob
+    include Sidekiq::Job
+
+    sidekiq_options retry: false
+
+    def perform(id)
+      user_verification = UserVerification.find(id)
+      MHV::UserAccount::Creator.new(user_verification:).perform
```

Comment:
We want to break the cache every time this job runs so we'll need to pass `break_cache: true` here